pkg/api/docs/definitions/correlations.go

```diff
@@ -31,3 +31,30 @@ type CreateCorrelationResponse struct {
 	// in: body
 	Body correlations.CreateCorrelationResponse `json:"body"`
 }
+
+// swagger:route DELETE /datasources/uid/{uid}/correlations/{correlationUid} correlations deleteCorrelation
+//
+// Delete a correlation.
+//
+// Responses:
+// 200: deleteCorrelationResponse
+// 401: unauthorisedError
+// 403: forbiddenError
+// 404: notFoundError
+// 500: internalServerError
+
+// swagger:parameters deleteCorrelation
+type DeleteCorrelationParams struct {
+	// in:path
+	// required:true
+	DatasourceUID string `json:"uid"`
+	// in:path
+	// required:true
+	CorrelationUID string `json:"correlationUid"`
```

Comment:
We would need to standardise the naming, since we decided dashboardUID would stay UID in capital, It is better to keep the same stype to `correlationUID`

Reply:
changed in 5481a344ee

also created https://github.com/grafana/grafana/pull/52836 to handle previously merged changes.